### PR TITLE
Resolved warnings

### PR DESCRIPTION
**Pull request by Neurosploit**
To get a step closer to a zero warning policy I tried to resolve most outstanding xunit and all IDE0009 warnings.
Also disabled the explicit var warning: IDE0008.

### CODE REVIEW

**Comment by bokobza** [comment]
I'll take it

File: Stratis.Bitcoin.Common/Stratis.Bitcoin.Common.csproj [hunk 1]
```diff
@@ -4,6 +4,10 @@
     <TargetFramework>netstandard1.6</TargetFramework>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <NoWarn>1701;1702;1705;IDE0008;</NoWarn>
```
Comment (bokobza) [inline Stratis.Bitcoin.Common/Stratis.Bitcoin.Common.csproj]: I can't see any warnings for CS1701, CS1702, CS1705. Also I don't think they need to be ignored should they show up.

**Comment by bokobza** [comment]
thanks for doing that @Neurosploit . I just have this one comment and then we're good to go.

File: Stratis.Bitcoin.Common/Stratis.Bitcoin.Common.csproj (hunk 1 above)
Comment (Neurosploit) [inline Stratis.Bitcoin.Common/Stratis.Bitcoin.Common.csproj]: I'll just leave this here. I think it is an issue in the sdk that will be fixed in version 2. https://github.com/dotnet/roslyn/issues/19640

**Pull request merged**